tests/Integration/Command/MeilisearchImportCommandTest.php

```diff
@@ -391,4 +391,47 @@ public function testAlias(): void
 
         self::assertSame(['meili:import'], $command->getAliases());
     }
+
+    public function testImportingIndexWithSwap(): void
+    {
+        for ($i = 0; $i <= 5; ++$i) {
+            $this->createPost();
+        }
+
+        $command = $this->application->find('meilisearch:import');
+        $commandTester = new CommandTester($command);
+        $return = $commandTester->execute([
+            '--swap-indices' => true,
+        ]);
+
+        $output = $commandTester->getDisplay();
+        $this->assertStringContainsString('Importing for index Meilisearch\Bundle\Tests\Entity\Post', $output);
+        $this->assertStringContainsString('Indexed a batch of '.$i.' / '.$i.' Meilisearch\Bundle\Tests\Entity\Post entities into _tmp_sf_phpunit__'.self::$indexName.' index ('.$i.' indexed since start)', $output);
+        $this->assertStringContainsString('Swapping indices...', $output);
+        $this->assertStringContainsString('Indices swapped.', $output);
+        $this->assertStringContainsString('Deleting temporary indices...', $output);
+        $this->assertStringContainsString('Deleted _tmp_sf_phpunit__'.self::$indexName, $output);
+        $this->assertStringContainsString('Done!', $output);
+        $this->assertSame(0, $return);
+
+        // Reset database and MS indexes
+        parent::setUp();
+
+        for ($i = 0; $i <= 5; ++$i) {
+            $this->createPost();
+        }
+
+        $command = $this->application->find('meilisearch:import');
+        $commandTester = new CommandTester($command);
```

Comment:
I'd split to separate test

Reply:
I've split it in two methods, is it better?

Reply:
Yes, just left one one question :)